R-packages/evalcast/R/target.R

```diff
@@ -51,7 +51,7 @@ get_target_period <- function(forecast_date, incidence_period, ahead) {
 #'
 #' @importFrom rlang .data
 #' @importFrom tibble enframe
-#' @importFrom dplyr mutate
+#' @importFrom dplyr mutate left_join
 #' @importFrom purrr pmap pmap_dfr
```

Comment:
And (while I'm at it) `pmap_lgl` here.

Reply:
Sure (this one's not actually in the NAMESPACE, so it's a good catch).

Reply:
And, I'm happy to leave it here, but just as a note, a quick grep shows that this function isn't actually used anywhere in the code yet.